packages/@aws-cdk/aws-kinesisfirehose/lib/delivery-stream.ts

```diff
@@ -0,0 +1,99 @@
+import * as elasticsearch from '@aws-cdk/aws-elasticsearch';
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import { IResource, RemovalPolicy, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnDeliveryStream } from './kinesisfirehose.generated';
+
+export enum DeliveryStreamType {
+  DIRECT_PUT = 'DirectPut',
+}
+
+export interface IDeliveryStream extends IResource {
+
```

Comment:
Let's add `streamArn` and `streamName` as attributes here (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-kinesisfirehose-deliverystream.html#aws-resource-kinesisfirehose-deliverystream-return-values).